doc/news/changes/incompatibilities/20240513Bangerth-2

```diff
@@ -0,0 +1,14 @@
+Changed: The Quadrature class used to have a constructor that took an
+integer argument. This was error prone because it was easy to accidentally
```

Comment:
```suggestion
Changed: The Quadrature class used to have a constructor that took an
integer argument. This was error-prone because it was easy to accidentally
```